File: examples/calculator-java/server/src/main/java/eth/likespro/lpfcp/calculator/ServerMain.java
package eth.likespro.lpfcp.calculator;

import eth.likespro.lpfcp.LPFCPJava;
import eth.likespro.lpfcp.ktor.Ktor;

public class ServerMain {
    public static void main(String[] args) {
        // Creating a calculator object like always
        Calculator calculator = new CalculatorImpl();

        // Starting LPFCP Server, which exposes all annotated calculator functions
        Ktor.INSTANCE.lpfcpServer(calculator, 8080)
                .start(true);
    }
}

File: examples/calculator-kotlin/server/src/main/java/eth/likespro/lpfcp/calculator/ServerMain.kt
package eth.likespro.lpfcp.calculator

import eth.likespro.lpfcp.ktor.Ktor.lpfcpServer

fun main() {
    // Creating a calculator object like always
    val calculator = CalculatorImpl()

    // Starting LPFCP Server, which exposes all annotated calculator functions
    lpfcpServer(processor = calculator, port = 8080)
        .start(wait = true)
}